guardian/support-dotcom-components
# Refactor choicecard code into common folder

## What does this change?
Choice card code currently lives in the `choiceCardsButtonBanner` component folder, but is used across several banner components. This PR copies the code into the `common` folder and updates the `designableBanner` and `momentTemplate` components to use that version of the code.

Work to clean up the `worldPressFreedomDay` and `choiceCardsButtonsBanner` components, which use localised choiceCard code, will be taken forward in a future PR. We expect that both of these components will be replaced by designable banners in due course.

Remaining banner components that use choice cards use the momentTemplate banner, and thus will use the code in the common folder.

Additionally, this PR removes local SVG `PaymentCards` code from the `designableBanner` and `momentTemplate` components and updates those components to use the existing `PaymentCards` component in the common folder.

## How to test
+ Test in Storybook and in CODE to make sure that all banners continue to work.
+ Check Chromatic snapshots to make sure these changes do not result in unexpected visual changes.
